packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -195,9 +195,13 @@ class DatePicker extends React.Component {
      */
     onWeekSelect: PropTypes.func,
     /**
-     * A callback function to execute when the mouse cursor leaves the element.
+     * **Deprecated**, A callback function to execute when the mouse cursor leaves the element. Resolves to `onDismiss`.
      */
     onClickOutside: PropTypes.func,
+    /**
+     * A callback function to execute when the mouse cursor leaves the element.
```

Comment:
Add `onDismiss(event)` so consumers know what parameter(s) will be sent back.

Reply:
Added here https://github.com/cerner/terra-framework/pull/1224/commits/22d489afc2983bcb40c258881a6ed2beab768874